go.mod1

```diff
@@ -7,12 +7,10 @@ require (
 	github.com/cznic/sortutil v0.0.0-20150617083342-4c7342852e65
 	github.com/cznic/strutil v0.0.0-20171016134553-529a34b1c186
 	github.com/cznic/y v0.0.0-20170802143616-045f81c6662a
-	github.com/pingcap/check v0.0.0-20181213055612-5c2b07721bdb
+	github.com/pingcap/check v0.0.0-20190102082844-67f458068fc8
 	github.com/pingcap/errors v0.11.0
-	github.com/pingcap/tidb v0.0.0-20181217070741-096bb68e6bef
+	github.com/pingcap/tidb v0.0.0-20180108134023-971629b9477a
```

Comment:
Why is the date 201<b>8</b>0108134023 🤔

Reply:
git commit hash is right, ref to https://github.com/pingcap/tidb/commit/971629b9477a9027f30e83f1fdb17120c0f2bec1 . strange

Reply:
it is because of the bug in `gopkg.in/stretchr/testify.v1@v1.3.0`